src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -373,8 +408,8 @@ public Response updateItem(UUID itemId, Item payload) throws Exception {
     String contactEmail = payload.getContactEmail();
     String contactPhone = payload.getContactPhone();
     String termsOfDelivery = payload.getTermsOfDelivery();
-    Boolean allowPurchaseContactSeller = payload.getPaymentMethods().isAllowContactSeller();
-    Boolean allowPurchaseCreditCard = payload.getPaymentMethods().isAllowCreditCard();
+    Boolean allowPurchaseContactSeller = payload.getPaymentMethods() != null ? payload.getPaymentMethods().isAllowContactSeller() : false;
+    Boolean allowPurchaseCreditCard = payload.getPaymentMethods() != null ? payload.getPaymentMethods().isAllowCreditCard() : false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=47&issues=AWZVH_tLBfefbO9HrBx5&open=AWZVH_tLBfefbO9HrBx5)

[ISSUE_KEY]: AWZVH_tLBfefbO9HrBx5